pkg/northbound/gnmi/set.go

```diff
@@ -386,3 +403,18 @@ func setChange(target string, version string, targetUpdates map[string]*change.T
 	}
 	return changeID, configName, false, nil
 }
+
+func validateChange(target string, version string, targetUpdates map[string]*change.TypedValue, targetRemoves []string) (change.ID, error) {
```

Comment:
again, because you don't store the change I think there's no point in returning it

Reply:
done